modules/05-firmware.yml

```
@@ -1,16 +1,31 @@
-name: input-and-locale
-type: apt
-source:
-  packages:
-  - firmware-linux
-  - firmware-linux-nonfree
-  - firmware-linux-free
-  - firmware-iwlwifi
-  - firmware-realtek
-  - firmware-atheros
-  - intel-microcode
-  - amd64-microcode
-  - b43-fwcutter
-  - firmware-b43-installer
-  - firmware-brcm80211
-  - firmware-sof-signed
+name: firmware-update
+type: shell
+commands:
+  - curl -O https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/snapshot/linux-firmware-20240513.tar.gz
+  - tar xzf linux-firmware-20240513.tar.gz
+  - rsync -avHAX --existing linux-firmware-20240513/ /usr/lib/firmware/
+  - rm linux-firmware-20240513.tar.gz
+  - rm -rf linux-firmware-20240513
+modules:
+- name: dependencies
+  type: apt
+  source:
+    packages:
+    - curl
+    - rsync
+- name: input-and-locale
```

Kinda curious is input and locale the apt name for this module?

Wouldn't something like:

```suggestion
- name: install-firmware
```

Be better?